### PR TITLE
Activity

### PR DESCRIPTION
**Pull request by ace0625**
This pull request is about activity & user profile.
We can ignore all of generated code except below files.

+95 −16  src/components/Activity/Activity.js
+123 −18  src/components/Activity/Row.js
+2 −0  src/components/App.js
+0 −1  src/components/Login.js
+5 −4  src/components/Main.js
+2 −1  src/components/SplashPage.js
+5 −1  src/components/UserList/Row.js
+86 −25  src/components/userProfile/UserProfile.js
+1 −1  src/utils/LoginUtil.js
+6 −0  src/utils/ServerUtil.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,11 @@
     "preset": "jest-react-native"
   },
   "dependencies": {
+    "install": "^0.8.1",
+    "npm": "^3.10.7",
     "react": "15.2.1",
     "react-native": "0.31.0",
+    "react-native-communications": "^2.1.0",
```
Comment (heartjigi) [inline package.json:17]: what do this component do?

Comment (ace0625) [reply package.json:17]: For built-in email.

File: src/components/userProfile/UserProfile.js [hunk 2]
```diff
@@ -25,41 +25,69 @@ class UserProfile extends Component {
...
+      ServerUtil.getMyProfile();
+    else
+      ServerUtil.getOthersProfile(this.props._id);
   }
 
   sendRequest() {
-
-    // TODO: Replace with server data
-    ServerUtil.sendMentoringRequest(this.state.id,  'I');
+    ServerUtil.sendMentoringRequest(this.state.id, 'I love ya');
```
Comment (heartjigi) [inline src/components/userProfile/UserProfile.js:90]: why don't U change this test msg

Comment (ace0625) [reply src/components/userProfile/UserProfile.js:90]: will change

**Comment by heartjigi** [comment]
LGTM

**Comment by heartjigi** [comment]
lgtm

**Pull request merged**